client/app/src/main/java/com/microsoft/research/karya/injection/RetrofitModule.kt

```diff
@@ -27,13 +28,30 @@ class RetrofitModule {
     @Provides
     @Reusable
     fun provideBaseUrl(): String {
-        return "http://192.168.0.105:4040"
+        return "https://karyabox2.eastus.cloudapp.azure.com/"
     }
 
     @Provides
     @Reusable
-    fun provideRetrofitInstance(baseUrl: String, converterFactory: GsonConverterFactory): Retrofit {
+    fun provideLoggingInterceptor(): HttpLoggingInterceptor {
+        return HttpLoggingInterceptor().setLevel(HttpLoggingInterceptor.Level.BODY)
```

Comment:
Might want to add some debug build checks here?

Reply:
Good idea :ok_hand: